packages/react-components/src/organisms/PeopleCard.tsx

```diff
@@ -32,6 +32,7 @@ const moveStyles = css({
   [`@media (min-width: ${tabletScreen.min}px)`]: {
     flexBasis: 'auto',
     order: -1,
+    lineHeight: `${lineHeight}px`,
```

Comment:
This should be inherited from the  `html` element already o.O

Reply:
Caption seems to have a line height specified within it making it hard to override.

Reply:
Oh, I didn't see there was a caption - this looks like it shouldn't be a caption, but normal text with styling adjustments. This likely generates illegal HTML, a `<figcaption>` without a `<figure>`.

Reply:
We may have to rethink the `Caption` component, but either way given that this deviates even from the `Caption` in styles, this probably isn't a usage of the Caption in design system terms